app/views/pages/service_manual.html.erb

```diff
@@ -0,0 +1,161 @@
+
+        <p>
+          Service Performance is a reporting platform that lets government measure the performance of transaction services.
+        </p>
+
+        <p>
+          Its main users are civil servants working to transform services. The platform is designed to help these users understand:
+        </p>
+        <ul class="list list-bullet">
+          <li>the demands faced by services - the volume of transactions they receive across different channels, and how often each channel is used and why</li>
+          <li>how well services are meeting their users' needs</li>
+          <li>how far the service has been digitised</li>
+        </ul>
+
+        <h2 class="heading-large" id="what-to-publish">
+          What do services need to publish?
+
+        </h2>
+        <p>
+          For the platform to function correctly, services need to publish their data to it. Data should be published every month; you’ll receive a reminder each time a new set of data is due, with a link to a webform where you can fill in your data.
+
+        </p>
+
+        <p>
+          The data should be published according to the following set of metrics:
+        </p>
+
+
+        <h4 class="heading-small">Total number of transactions received</h4>
+        <p>
+          This KPI helps users gauge the overall demand for the service.
+        </p>
+
+        <h4 class="heading-small">Total number of transactions received, split by channel</h4>
+        <p>
+          This KPI helps users see which channels are used the most.
+        </p>
+        <p>Channels include:</p>
+        <ul class="list-bullet">
+          <li>online</li>
+          <li>phone</li>
+          <li>paper form</li>
+          <li>face-to-face</li>
+          <li>other</li>
+        </ul>
+
+        <h4 class="heading-small">Number of transactions processed</h4>
+        <p>
+          This KPI gives users an indication of how well the service is meeting users’ needs.
+        </p>
+
+        <h4 class="heading-small">Number of transactions processed with the user’s intended outcome</h4>
+        <p>
+          This KPI gives users more information about how well the service is meeting users’ needs.
+        </p>
+
+        <h4 class="heading-small">Total number of phone calls received</h4>
+        <p>
+          This KPI shows how much the service's contact centre is used.
+        </p>
+
+        <h4 class="heading-small">Number of phone calls received, split by reasons for calling</h4>
+        <p>
+          This KPI shows why calls are made and gives an indication of how the number of calls can be reduced.
+        </p>
+        <p>
+        Reasons include:</p>
+        <ul class="list-bullet">
+          <li>to perform a transaction</li>
+          <li>to get information</li>
+          <li>to chase progress</li>
+          <li>to challenge a decision</li>
+          <li>Other</li>
```

Comment:
Should this be lowercase, to be consistent with the rest of this list and the previous breakdown?

Reply:
Oops. Will fix.